syncer/syncer.go

```diff
@@ -1294,7 +1291,9 @@ func (s *Syncer) syncDML(
 	idx := 0
 	count := s.cfg.Batch
 	jobs := make([]*job, 0, count)
-	tpCnt := make(map[opType]int64)
+	// db_schema->db_table->opType
+	tpCnt := make(map[string]map[string]map[opType]int64)
+	queueID := strings.Split(workerLagKey, "lag_")[0]
```

Comment:
also recommend that use the index (number) not string to organize lags of different queues.

Reply:
this PR is already too large,  Maybe next PR i will do this change with https://github.com/pingcap/dm/issues/1879